### DISCUSSION TITLE
在译文中是否应该添加标点符号？

### DISCUSSION THREAD

**Discussion by ahdg6** [Tartaros-Gamer/GriefDefenderSinicization#2] [General]
https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md
我认为还是要的
![图片](https://user-images.githubusercontent.com/36436808/130011316-47c42875-dcf6-4711-b7b8-deeee272452b.png)

**Comment by pys078** [comment]
文本量小的情况下添加过多的标点符号可能导致文本显示效果不佳
应参照英文版本中不添加结尾符号的做法

**Reply by ahdg6** [reply]
可以，但像示例那种断句还是要的吧，空格不是万能的，专有名词之间也会要空格的
